_layouts/agenda.html

```diff
@@ -4,12 +4,20 @@
 <div class="post">
   <header class="post-header">
     <h1 class="post-title">Agenda for {{ page.date | date: "%B %-d, %Y" }}</h1>
+    <h6 class="masthead">BAKERSFIELD COLLEGE</h6>
```

Comment:
If this is all caps just for presentation preference, it might be better done with CSS.